users/templates/user_profile/base_user_profile.html

```diff
@@ -6,6 +6,7 @@
             <ul class="collection">
                 <li class="collection-item {% if request.path == "/user/profile/" %} usermenu-active {% endif %}"><a href="{% url 'edit_profile' %}">User profile</a></li>
                 <li class="collection-item {% if request.path == "/user/settings/password/" %} usermenu-active {% endif %}"><a href="{% url 'password' %}">Change password</a></li>
+                <li class="collection-item {% if request.path == "/user/settings/password/" %} usermenu-active {% endif %}"><a href="{% url 'send_invitation' %}">Send invitation</a></li>
```

Comment:
"if request.path == "/user/settings/password/""... change this to the right url (this is to activate the menu)

Reply:
What's the best option? "/send_invitation/" or "/user/send_invitation" or other?

Reply:
hmm... /user/send_invitation/ sounds good!

Reply:
Done!